public/index.html

```diff
@@ -175,8 +175,30 @@ <h4>Include links to privacy policy of Third Party Services <small>(Scroll for m
             </div>
           </section>
         </div>
+
         <div class="mdl-layout__tab-panel" id="privacypolicy">
           <section class="section--center mdl-grid">
+
+            <div class="mdl-cell mdl-cell--12-col mdl-shadow--2dp" id="fc-container">
+              <div id="fc-container-text">
+                <div style="color:#302428;">Free privacy policy hosting + website for {{appName}}.</div>
+                <div style="font-size:0.7em;color:#979193;">Powered by <a href="https://flycricket.com/?utm_source=partner&utm_medium=website&utm_campaign=privacy_generator" target="_blank">Flycricket</a></div>
```

Comment:
Shouldn't the referrer be added here too? 🤔

Reply:
Yes - updated, good catch. I figured this isn't really a CTA but we should have anyway.